docs/docs/configuration.md

```diff
@@ -52,6 +52,8 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.parquet.bloom-filter-enabled.column.col1       | (not set)                   | Hint to parquet to write a bloom filter for the column: 'col1'                                                                                                                                    |
 | write.parquet.bloom-filter-max-bytes                 | 1048576 (1 MB)              | The maximum number of bytes for a bloom filter bitset                                                                                                                                             |
 | write.parquet.bloom-filter-fpp.column.col1           | 0.01                        | The false positive probability for a bloom filter applied to 'col1' (must > 0.0 and < 1.0)                                                                                                        |
+| write.parquet.stats-enabled.default                  | true                        | Controls whether to collect parquet column statistics when not specified on column level, currently not configurable [parquet-java#3189](https://github.com/apache/parquet-java/issues/3188)      |
+| write.parquet.stats-enabled.column.col1              | (not set)                   | Controls whether to collect parquet column statistics for column 'col1'; true or false                                                                                                            |
```

Comment:
we can probably remove `true or false` at the end, similar to other boolean typed table properties